lib/src/browser.dart

```diff
@@ -40,6 +40,10 @@ class Browser {
 
   Version _version;
 
+  void reset() {
```

Comment:
so do we actually need to call this in tests or should it just be called by the example app?